quickstart/conf/regorule.yaml

```diff
@@ -0,0 +1,90 @@
+---
+apiVersion: opa.stackable.tech/v1alpha1
+kind: RegoRule
+metadata:
+  name: trino2
+spec:
+  rego: |
```

Comment:
This would be deployed by the trino operator. Im fine with keeping it in here though.